herd/AArch64Annot.ml

```diff
@@ -18,6 +18,7 @@
 type t =
   A | XA | L | XL | X | N | Q | XQ | NoRet | S
   | NTA (* Non-Temporal, avoid clash with NT in AArch64Base *)
+  | SysDirect (* Direct accesses to system registers *)
```

Comment:
Rather than adding a new annotation can we please use
`Exp | NExp`
That is already there for memory accesses? Then in the cat file we can use Exp to mean Direct and NExp to mean indirect.

That way you won't have to another type here and you will only need to change `herd/AArch64Sem.ml`